### PR TITLE
Organization

### PR DESCRIPTION
**Pull request by mishalhossin**

### CODE REVIEW

File: main.py
Comment (mak448a) [inline main.py:1]: Nice! Adding classes, reorganizing, and restructuring. Anyway, looks great and I’ll try it out later. Thank you for the PR

Comment (mishalhossin) [reply main.py:1]: Also removed file for linejunk

Comment (mishalhossin) [reply main.py:1]: i mean the file

Comment (mishalhossin) [reply main.py:1]: > Nice! Adding classes, reorganizing, and restructuring. Anyway, looks great and I’ll try it out later. Thank you for the PR

I was gonna do pr for all the cogs but didnt had time to do so

Comment (mak448a) [reply main.py:1]: > > Nice! Adding classes, reorganizing, and restructuring. Anyway, looks great and I’ll try it out later. Thank you for the PR
> 
> I was gonna do pr for all the cogs but didnt had time to do so

Yeah that’s fine I already did that 👍🏼

**Comment by mishalhossin** [comment]
Dont forget to merge this later when its good for production

**Comment by mak448a** [comment]
> Dont forget to merge this later when its good for production

I will 👍🏻

**Comment by mak448a** [comment]
Oh also if you want to talk over discord: https://discord.gg/QyVhcdAfre

**Comment by mak448a** [comment]
Uh oh just tested and line junk doesn't work, fixing...

**Comment by mak448a** [comment]
The line junk also had invisible characters in it so I just put it into a string

**Comment by mak448a** [comment]
Ok merging....

**Comment by mishalhossin** [comment]
> Oh also if you want to talk over discord: https://discord.gg/QyVhcdAfre

Invalid

**Pull request merged**